modules.d/50plymouth/module-setup.sh

```diff
@@ -1,27 +1,77 @@
 #!/bin/bash
 
 pkglib_dir() {
-    local _dirs="/usr/lib/plymouth /usr/libexec/plymouth/"
+    local _dirs="/usr/lib/plymouth /usr/lib64/plymouth /usr/libexec/plymouth"
```

Comment:
Who is incorrectly shipping this in `/usr/lib64/plymouth`? I don't want to support that.